tests/unit/test_utils/test_typing.py

```diff
@@ -3,10 +3,9 @@
 from __future__ import annotations
 
 from sys import version_info
-from typing import Any, Dict, Generic, List, Optional, TypeVar, Union
+from typing import Annotated, Any, Dict, Generic, List, Optional, TypeVar, Union  # noqa: UP035
```

Comment:
i dont get the UP035

Reply:
We test these annotations here (`Dict`, `List`, etc.). And while they are not required anymore on > 3.8, they are still available and we want to support them, so it's important that we test for those. Ruff though wants to upgrade them away, so we prevent it from doing so